src/api/api.py

```
@@ -139,63 +162,171 @@ def __init__(self, database: Database, namespace: str = 'default'):
+            body: Payload describing the analysis to create (image, registry
+                credentials, Kong token, etc.).
+
+        Returns:
+            A mapping of ``{analysis_id: status}`` for the newly started run.
+
+        Raises:
+            HTTPException: 500 on any downstream failure (details in logs).
+        """
         try:
             return create_analysis(body, self.database)
         except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)